src/game_state/Collectible.h

```diff
@@ -16,12 +17,15 @@ class Collectible : public GameObject {
 
   void SetCollected();
   void SetUncollected();
+  void IncrementTicksCollected();
   bool GetCollected() const;
+  int GetTicksCollected();
```

Comment:
I'd rather go for setting a tick when it was collected and determining how many ticks have passed each render pass and using a number of seconds with TICKS_PER_SECOND to convert the ticks to units of animation, but this is ok. You have the right idea for using the timing system :ok_hand: